sdk/unity/xr_unity_plugin/cardboard_xr_unity.h

```diff
@@ -17,6 +17,7 @@
 #define CARDBOARD_SDK_UNITY_XR_UNITY_PLUGIN_CARDBOARD_XR_UNITY_H_
 
 #include <memory>
+#include "include/cardboard.h"
```

Comment:
taking into account the comments above, this shouldn't be required anymore. Furthermore, the purpose of the CardboardApi class is to hide the inner workings of the Cardboard SDK, avoiding the inclusion of `cardboard.h` here.